lib/inject.rb

```diff
@@ -0,0 +1,26 @@
+class Array
+
+  def my_inject(*args, &block)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for my_inject is too high. [21.47/15]